### PR TITLE
fix: correct app component template for selector atlasmap-dev-root

### PR DESCRIPTION
**Pull request by pleacu**
in support of vscode.  WIll need to get migrated to 1.43.x
Fixes: #1704

### CODE REVIEW

File: ui/src/app/app.component.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'atlasmap-dev-root',
-  template: '<router-outlet></router-outlet>',
```
Comment (igarashitm) [inline ui/src/app/app.component.ts]: This disables multiple mapping definition support, DataMapperExampleHostComponent is now delivered through angular router https://github.com/atlasmap/atlasmap/blob/master/ui/src/app/app-routing.module.ts#L12

As it has been working for browsers without any URL change, I suspect root cause is in vscode-atlasmap and/or vscode webview
https://github.com/jboss-fuse/vscode-atlasmap

@AbbassFaytaroony do you have chance to look into this problem? I'll also look into.

Comment (igarashitm) [reply ui/src/app/app.component.ts]: It's getting `Unhandled Navigation Error` in @angular/core when it tries to handle `<router-outlet>` via vscode WebView.

Comment (igarashitm) [reply ui/src/app/app.component.ts]: This is replacing `<base href=` to the absolute URL, but it still doesn't work
https://github.com/jboss-fuse/vscode-atlasmap/blob/master/src/atlasMapWebView.ts#L95

**Comment by pleacu** [comment]
Corrected in vscode-atlasmap

**Pull request closed without merging**